### PR TITLE
Turn navbar into accordion if the screen is small

### LINKED ISSUE #359: Fix navbar screen shrink

### PR DESCRIPTION
**Pull request by karlmolina**
closes #359 

image

### CODE REVIEW

File: client/src/components/ui/Accordion.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Component } from 'react';
+
+class Accordion extends Component {
```
Comment (sambehrens) [inline client/src/components/ui/Accordion.js]: this looks like a very important component

**Comment by sambehrens** [comment]
Screen Shot 2019-04-12 at 7 18 24 AM

**Comment by sambehrens** [comment]
Should links stretch across the whole accorian div so that you don't have to be as accurate with your touch since this is really going to be most used on mobile?

File: client/src/components/ui/Accordion.js (hunk 1 above)
Comment (karlmolina) [reply client/src/components/ui/Accordion.js]: lol

**Comment by sambehrens** [comment]
Is this a problem? You could just give them a little more padding on the right
Screen Shot 2019-04-12 at 9 04 35 AM

File: client/src/components/ui/Accordion.js (hunk 1 above)
Comment (karlmolina) [reply client/src/components/ui/Accordion.js]: maybe i'll extract it later

Comment (sambehrens) [reply client/src/components/ui/Accordion.js]: what its not even used

**Review by sambehrens — APPROVED** [review]

**Pull request merged**